src/main/java/com/mg105/entities/Doorway.java

```diff
@@ -0,0 +1,35 @@
+package com.mg105.entities;
+
+public class Doorway {
+    private final RoomPosition position;
+    private final Room nextRoom;
+
+    /**
+     * Create a doorway with its position in the room and the next room.
+     *
+     * @param position the in-room position of this doorway.
+     * @param nextRoom the room that should be travelled to if this doorway is entered.
+     */
+    public Doorway(RoomPosition position, Room nextRoom) {
+        this.position = position;
+        this.nextRoom = nextRoom;
+    }
```

Comment:
nice way to access the room and next-room, but variable type for these instance variables should depends on the way we store the rooms.